app/konnect/declarative-config.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Managing a Kong-hosted control plane with declarative config
```

Comment:
Why do we say Kong-host control plane in the title but Konnect cloud control plane in the paragraph below?